lib/rouge/lexers/console.rb

```diff
@@ -48,9 +48,9 @@ def allow_comments?
 
       def prompt_prefix_regex
         if allow_comments?
-          /^[^<#]*?/
+          /[^<#]*?/m
```

Comment:
NOTE: `^` is redundant because [the parent regex builder](https://github.com/jneen/rouge/blob/d27196e754054b3438e1690e51230b4583b7c20b/lib/rouge/lexers/console.rb#L26) specifies it.